.github/workflows/reusable-image-and-iso-arm-generic.yaml

```diff
@@ -35,15 +47,21 @@ jobs:
         run: |
           sudo iptables -I INPUT -s 169.254.169.254 -j DROP
           sudo iptables -I OUTPUT -d 169.254.169.254 -j DROP
-      - name: Build iso  🔧
-        env:
-          IMAGE: "quay.io/kairos/core-${{ inputs.flavor }}-arm-generic"
-          TAG: "master"
+      - name: Build iso 🔧
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
         run: |
-          earthly -P +all-arm-generic --FLAVOR=${{ inputs.flavor }} --IMAGE=$IMAGE:$TAG --ISO_NAME=kairos-${{ inputs.flavor }}-arm-generic-$TAG
+          earthly --allow-privileged +all-arm-generic \
+              --FAMILY=${{ inputs.family }} \
+              --FLAVOR=${{ inputs.flavor }} \
+              --FLAVOR_RELEASE=${{ inputs.flavor_release }} \
+              --BASE_IMAGE=${{ inputs.base_image }} \
+              --MODEL=${{ inputs.model }} \
+              --VARIANT=core \
+              --IMG_COMPRESSION=${{env.IMG_COMPRESSION}}
       - name: Push to quay
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        #if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        # disable artifact upload for now
```

Comment:
what for? Either we disabled it for good or we should write here when this is supposed to be enabled back. If we don't want "master" artifacts, let's just remove this from here and check that the release pipeline will produce something when we tag a new release.

Reply:
mainly because I have no idea under what repo should this go or how xDD

Reply:
ok we can fix it later but the flags we pass to the dockerfile/earthly are determining the artifact name. These are the same args we pass to the naming.sh script. `naming.sh` will tell you where it should be pushed if you instruct it to give you a "container_artifact_name".